Scroller/EditableWrapper.js

```diff
@@ -697,10 +699,23 @@ const EditableWrapper = (props) => {
 				if (!scrollContainerRef.current.contains(nextTarget)) {
 					const orders = finalizeOrders();
 					finalizeEditing(orders);
+				} else if ((is('left', keyCode) || is('right', keyCode)) && selectedItem) {
+					moveItemsByKeyDown(ev);
+				} else if (is('down', keyCode) && selectedItem) {
+					completeEditingByKeyDown();
+				} else if (is('up', keyCode) && nextTarget.getAttribute('role') !== 'button') {
+					setPointerMode(false);
+					Spotlight.focus(nextTarget);
+				}
+			} else if (is('enter', keyCode)) {
+				if (selectedItem) {
+					completeEditingByKeyDown();
+				} else {
+					startEditing(focusedItem);
```

Comment:
How about
```suggestion
				} else
					handleKeyDownCapture(ev);
				}
			} else if (is('enter', keyCode)) {
				handleKeyDownCapture(ev);
```

Reply:
After testing, I found that the `handleKeyDownCapture` function does not work as we expect because the ev.target is the `body` element, not each item. Therefore, it seems difficult to change it to use `handleKeyDownCapture`.

Reply:
T.T

Reply:
I'll add some inline comments.